web-ng/src/app/components/layer-list-item/layer-list-item.component.ts

```diff
@@ -139,13 +137,9 @@ export class LayerListItemComponent implements OnInit, OnDestroy {
     );
   }
 
-  initLayerItemPermission(): void {
-    const project = this.projectService.getCurrentProject();
-    if (!project) {
-      return;
-    }
-    const acl = this.projectService.getProjectAcl(project);
-    this.canCustomizeLayer = this.authService.canManageProject(acl);
+  canShare(): boolean {
+    const acl = this.projectService.getCurrentProjectAcl();
+    return acl ? this.authService.canManageProject(acl) : false;
```

Comment:
I think this can be simplified as:
```
return acl && this.authService.canManageProject(acl);
```

Reply:
@gino-m , in the latest revision canShare() method doesn't exist. Thank you :)